NEWS.md

```diff
@@ -1,3 +1,39 @@
+Printrun - 2.0.0
+====================
+
+Stable release supporting Python 3.10 and wxPython 4.2.
+
+See changes in previous pre-releases below for a full change log from previous
+stable release.
+
+### New Features
+
+ - Dropped support for Windows 32-bit executable (0c296ba)
+ - Added PrintrunGTK3 sub-module. Windows libraries (0c296ba)
```

Comment:
I dont understand this line without looking at the commit.

Reply:
With wxPython 4.2 only x64 support is available.
projectlayer.py  uses libraries form GTK3.  GTK3 isn't part of Windows, therefore we need to add the missing libraries manually to the binary file.  (only a subset of dll's as we don't need the hole package)